alpha/alpha-server/src/test/java/org/apache/servicecomb/pack/alpha/server/cluster/master/ClusterLockServiceTest.java

```diff
@@ -60,37 +65,68 @@
   ClusterLockService clusterLockService;
 
   @MockBean
-  private MasterLockRepository masterLockRepository;
+  private MasterLockEntityRepository masterLockEntityRepository;
 
-  @Test(timeout = 10000)
-  public void testClusterNodeType() throws InterruptedException {
-    //node type is master
-    clusterLockService.setMasterLock(null);
+  @Autowired
+  MasterLockRepository masterLockRepository;
+
+  @Before public void before(){
```

Comment:
It's strange for me to see you reset the test status.
If the masterLockEntityRepository is mocked, it's easy for us to reset it.
Is there anything that I'm missing?

Reply:
Use variable lockInitialization in the doLock method of AbstractLockProvider to avoid executing the initLock method again. InitLock method cannot be executed in unit test

Reply:
Thx for the explain, I will add some comment on the before method shortly.